pkg/cri/resource-manager/policy/builtin/memtier/cache.go

```diff
@@ -86,11 +88,13 @@ func (ccg *cachedGrant) ToGrant(policy *policy) (Grant, error) {
 	}
 
 	g := newGrant(
+		nil, // FIXME: where to get the request corresponding to this grant?
```

Comment:
It would be nice to avoid having to have that here. It's easily doable (we could save the request in the cached grant) but I really think it should not be needed (more on that later).